docker/docker-compose.centos.yaml

```diff
@@ -0,0 +1,31 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: netty-tcnative-centos:default
+    build:
+      context: .
+      dockerfile: Dockerfile.centos
+
+  common: &common
+    image: netty-tcnative-centos:default
+    depends_on: [runtime-setup]
+    volumes:
+      - ~/.ssh:/root/.ssh
+      - ~/.gnupg:/root/.gnupg
+      - ..:/code
+    working_dir: /code
+
+  build:
```

Comment:
test?

Reply:
not sure... I thought build would be more fitting as it is really compiling everything and package it.